src/cascade/input_data/db/csmr.py

```diff
@@ -60,16 +61,19 @@ def _gbd_process_version_id_from_cod_version(cod_version):
 def _get_csmr_data(execution_context):
 
     cause_id = execution_context.parameters.add_csmr_cause
-    parent_loc = execution_context.parameters.location_id
 
     keep_cols = ["year_id", "location_id", "sex_id", "age_group_id", "val", "lower", "upper"]
 
     process_version_id = _gbd_process_version_id_from_cod_version(execution_context.parameters.cod_version)
 
+    location_hierarchy = get_location_hierarchy_from_gbd(execution_context)
+    location = location_hierarchy.get_node_by_id(execution_context.parameters.location_id)
+    location_and_children = [d.id for d in location.children] + [location.id]
```

Comment:
And third example of same pattern.

Reply:
Yes, you are right. I was avoiding thinking about how to clean that up but it's silly to have the repeated code.